packages/ComponentLibrary/src/components/Drawer/index.tsx

```diff
@@ -116,6 +124,24 @@ const Drawer: React.FC<DrawerProps> = ({ windowId, items = [], logo, title, onCl
     [items, onClick],
   );
 
+  const handleReportClick = useCallback(
+    (reportId: string) => {
+      if (onReportClick) {
+        onReportClick(reportId);
```

Comment:
Is it possible to remove this callback and use onReportClick directly, without wrapping it?